test_suite.sh

```diff
@@ -164,11 +164,13 @@ if [[ "${cpuinfo}" =~ (Core\(s\) per socket:[^0-9]*([0-9]+)) ]]; then
 else
     fatal_error "Failed to get the number of cores per socket for the current test hardware with lscpu."
 fi
-cgroup_v1_mem_limit="/sys/fs/cgroup/memory/$(</proc/self/cpuset)/memory.limit_in_bytes"
-cgroup_v2_mem_limit="/sys/fs/cgroup/$(</proc/self/cpuset)/memory.max"
+cgroup_v1_mem_limit="/hostsys/fs/cgroup/memory/$(</proc/self/cpuset)/memory.limit_in_bytes"
+cgroup_v2_mem_limit="/hostsys/fs/cgroup/$(</proc/self/cpuset)/memory.max"
```

Comment:
This path probably makes only sense if run in a very specific environment (e.g., testing software built for EESSI). While this is fine, how about checking whether `/sys` or `/hostsys` is available and use that?

If there would be a comment that explains what `/hostsys` is and how it is made available, it might make debugging a little easier.

Reply:
Yes, we bind-mount this additional path in `bot/test.sh`. You're absolutely right about the commenting part: I'll make that clear. 

Regarding a fallback on `/sys`, I'm not sure if we want to do that. If `/hostsys` isn't there, it means the bind-mount failed / was not executed. I'd probably prefer there to be a hard error, than a silent success here that maybe extracts the wrong amount of memory.

Reply:
Ok, added the description now